pom.xml

```
@@ -5,14 +5,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.flowingcode.vaadin.addons</groupId>
-    <artifactId>template-addon</artifactId>
+    <artifactId>date-time-range-picker-addon</artifactId>
     <version>1.0.0-SNAPSHOT</version>
-    <name>Template Add-on</name>
-    <description>Template Add-on for Vaadin Flow</description>
+    <name>DateTimeRangePicker</name>
```

```suggestion
    <name>Date Time Range Picker Add-on</name>
```

FlowingCode/TechnicalDocs#91
https://github.com/FlowingCode/TechnicalDocs/tree/master/addons#naming-convention